### PR TITLE
Bookstore: add reducers and actions

### PR DESCRIPTION
**Pull request by Olivier-Kango**
# Bookstore: add reducers and actions

### Hi there 👋,
In this particular project, I :

- Add Redux Toolkit(`npm install react-redux @reduxjs/toolkit`).
- Structure my application files using a "feature folder" approach and use the [*ducks* pattern](https://github.com/erikras/ducks-modular-redux) for my Redux files. My files/folder structure is look like this:
    ```    
    ...
    /src
        |---/components
        |---/redux
            |--- /books
                    | books.js
            |--- /categories
                    | categories.js
            | configureStore.js
        | index.js
    ``` 
- Write my book's actions and reducer (in `/src/redux/books/books.js`) using the [*ducks* pattern](https://github.com/erikras/ducks-modular-redux). 
    - Define *action types* for adding and removing a book.
    - Set the initial state to be an empty array of books.
    - Export *Action Creators* for my *actions*.
    - Write my reducer and export it as default.
        - Define state changes for the actions that I created.
        - In case of unknown action - return the current state.
- Write my categories actions and reducer (in `/src/redux/categories/categories.js`) using the [*ducks* pattern](https://github.com/erikras/ducks-modular-redux). 
    - Define an *action type* for checking the status.
    - Set the initial state to be an empty array of categories.
    - Export *Action Creators* for my *actions*.
    - Write your reducer and export it as default.
        - For the check status action return a string "Under construction".
        - In case of unknown action - return the current state.

- Configure the Redux Store (`/src/redux/configureStore.js`):
    - Importing the necessary methods from Redux Toolkit.
    - Combine both reducers into a root reducer by using  `configureStore` function.

### CODE REVIEW

File: src/redux/configureStore.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import { createStore, combineReducers, applyMiddleware } from 'redux';
```
Comment (codecaiine) [inline src/redux/configureStore.js]: - [ ] From the project requirement you should Import the necessary methods from Redux Toolkit like Imports `configureStore `from `@reduxjs/toolkit`.  Please fix it :smile: 
Screenshot from 2022-11-14 22-12-04

File: README.md [hunk 2]
```diff
@@ -67,4 +67,4 @@ Give a ⭐️ if you like this project!
 
 This project is [MIT](./LICENSE) licensed.
 
-_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
\ No newline at end of file
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
```
Comment (codecaiine) [inline README.md]: - [ ] [OPTIONAL] Please remove this unnecessary information from your readme to make it more professional :smile: 
- [ ] [OPTIONAL] The link to your licence is broken , please fix it in order to make your project more professional :smile:

**Review by codecaiine — CHANGES_REQUESTED** [review]
Hi @Olivier-Kango, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- Good project implementation :heavy_check_mark: 
- No linters errors :heavy_check_mark: 
- Descriptive PR

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@codecaiine) in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 3]
```diff
@@ -62,9 +62,3 @@ Give a ⭐️ if you like this project!
 - Hat tip to anyone whose code was used
 - Inspiration
 - etc
-
-## 📝 License
-
-This project is [MIT](./LICENSE) licensed.
-
```
Comment (OLIPLICHE) [inline README.md:69]: # Optional:
- The previous reviewer said, " The link to your license is broken, fix it". He asked you to fix it and not to be removed it. 
- Kindly fix MIT, Make your project more professional.

**Review by OLIPLICHE — APPROVED** [review]
Hi @Olivier-Kango , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Highlights
- Implemented the change from the previous review. 👏 👏
- Everything is working fine👏 👏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**